mono/mini/TestHelpers.cs

```diff
@@ -0,0 +1,31 @@
+using System;
```

Comment:
The thing is we use the code both in mono tests and in bcl tests, so either way we would need to include from one side to the other. In the end, I added the file also in test-helpers as a symlink to the one in mini, to avoid duplicated code in the repo and to indicate that is logically also part of bcl.

Ideally I think we should have a Mono.TestHelpers assembly in mcs/class, where we can put all helpers, which we can reference in all test assemblies, whether they are from bcl or mono. But I don't know how much change that requires.